client/providers/OmniChannelProvider.tsx

```diff
@@ -96,6 +96,7 @@ const OmnichannelEnabledProvider: FC = ({ children }) => {
 		setContextValue((context) => ({
 			...context,
 			agentAvailable: user?.statusLivechat === 'available',
+			...routeConfig && { routeConfig },
```

Comment:
```suggestion
```

This isn't necessary anymore.

Reply:
@rafaelblink I was not able to see the `Move to queue` button due to because the `routeConfig` property was getting set up here :top:. 

This was the check that was failing -> https://github.com/RocketChat/Rocket.Chat/pull/21285/files#diff-7ea22d1f2dc14b7eb09df59074efe4d9a43c96cce7e9008e92e9b792fe13c56cR201

Reply:
Can you please verify if that's not the case anymore? If so, I'll revert the change

Reply:
https://github.com/RocketChat/Rocket.Chat/blob/develop/client/views/room/Header/Omnichannel/QuickActions/QuickActions.tsx#L180

@murtaza98 this is what @rafaelblink meant.

Reply:
@renatobecker I made this change because the `Move Queue` button wasn't visible in the header when I have Manual Selection Routing Method selected

**Actual Behaiour**
![image](https://user-images.githubusercontent.com/34130764/112425544-4475a380-8d5c-11eb-93f8-b5fbc9886673.png)

**Expected Behaviour**
![image](https://user-images.githubusercontent.com/34130764/112425618-64a56280-8d5c-11eb-84b8-00624d5065fa.png)

Reply:
Yeah, I got your point, that's the reason whe're sharing with you the right way to get the information you need, that's through the Omnichannel contexts. You just need to import it use it properly.

https://github.com/RocketChat/Rocket.Chat/blob/develop/client/contexts/OmnichannelContext.ts#L22

Reply:
@renatobecker We are already using the method [here](https://github.com/RocketChat/Rocket.Chat/pull/21285/files#diff-7ea22d1f2dc14b7eb09df59074efe4d9a43c96cce7e9008e92e9b792fe13c56cR196)

The issue is that when the Client is calling server to get this info over [here](https://github.com/RocketChat/Rocket.Chat/pull/21285/files#diff-3cb80e6d6d29b75c26f675e0c04a803fc66185b5b021515737836005c2c3e76fR87), its not saving the `routeConfig` property into the context, due to which we aren't able to access this property over [here](https://github.com/RocketChat/Rocket.Chat/pull/21285/files#diff-7ea22d1f2dc14b7eb09df59074efe4d9a43c96cce7e9008e92e9b792fe13c56cR201)